spellingbee/model/hint.py

```diff
@@ -150,8 +150,29 @@ def countWordsCheck(obj: puzzle.Puzzle) -> int:
     # Parameters:
     #   None
     ############################################################################
-    def numPangrams() -> int:
-        pass
+    def numPangrams(self) -> int:
+        ulString = puzzle.getUniqueLetters()
+        # SQLite Connections
+        wordDict = sqlite3.connect("spellingbee/model/wordDict.db")
+
+        # Used to execute SQL commands
+        wordDictC = wordDict.cursor()
+        # Grabs a random baseword from the list
+        wordDictC.execute(
+            """ SELECT COUNT(fullWord)
+                        FROM pangrams
+                        WHERE uniqueLetters LIKE %
+                        '""" + ulString + "';"
+        )
+        # catch return from querey
+        resultResult = wordDictC.fetchone()
+        
+
+        # close DB
+        wordDict.commit()
+        wordDict.close()
+        return resultResult
```

Comment:
heads up, this will return a tuple of ([COUNT],), you probably want to unzip this tuple if you're trying to return an integer